src/js/leaflet.storage.js

```diff
@@ -554,38 +559,55 @@ L.Storage.Map.include({
             shortUrl.value = this.options.shortUrl;
         }
         L.DomUtil.create('hr', '', container);
-        L.DomUtil.add('h4', '', container, L._('Download raw data'));
+        L.DomUtil.add('h4', '', container, L._('Download data'));
         var typeInput = L.DomUtil.create('select', '', container);
         typeInput.name = 'format';
-        L.DomUtil.add('small', 'help-text', container, L._('Only visible features will be downloaded.'));
+        var exportCaveat = L.DomUtil.add('small', 'help-text', container, L._('Only visible features will be downloaded.'));
+        exportCaveat.id = "export_caveat_text";
+        L.DomEvent.on(typeInput, 'change', function () {
+            if (typeInput.value == "umap") {
+                exportCaveat.style.visibility = "hidden";
```

Comment:
Why not using `display` instead? IIRC, visibility will keep the element space used.